experiments/calibration/run_calibration.jl

```diff
@@ -68,14 +68,20 @@ if !isfile(obs_path)
     touch(joinpath(obs_output_dir, "parameters.toml"))
     CAL.forward_model(0, 0)
     observations = Vector{Float64}(undef, 1)
-    observations .= process_member_data(SimDir(joinpath(obs_output_dir, "amip_config/output_active/clima_atmos")))
+    observations .= process_member_data(SimDir(joinpath(obs_output_dir, "model_config/output_active/clima_atmos")))
     JLD2.save_object(obs_path, observations)
 end
-
-# Initialize experiment data
-@everywhere observations = JLD2.load_object(obs_path)
-
-eki = CAL.calibrate(CAL.WorkerBackend, ensemble_size, n_iterations, observations, noise, prior, output_dir)
+observations = JLD2.load_object(obs_path)
+@show observations
```

Comment:
Is this `show` meant to be here, or was it for debugging?

Reply:
It was for debugging, but it would be nice to keep around to debug quickly from a buildkite log if this fails in the future.

Reply:
Okay we can keep it!